### PR TITLE
added money door plugin

### PR DESCRIPTION
**Pull request by tvuong123**
test

### CODE REVIEW

**Comment by oldstudent1997** [comment]
I see it

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+import edu.cmu.cs.cs214.rec10.framework.core.Player;
+
+import java.util.*;
+
+public class MoneyDoorPlugin implements GamePlugin {
```
Comment (oldstudent1997) [inline src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java:10]: nice code

Comment (oldstudent1997) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java:10]: good style

**Review by oldstudent1997 — APPROVED** [review]

File: src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java (hunk 1 above)
Comment (tvuong123) [reply src/main/java/edu/cmu/cs/cs214/rec10/plugin/MoneyDoorPlugin.java:10]: thanks man

**Pull request merged**